lib/color-formatters.js

```diff
@@ -7,11 +7,13 @@
 const moment = require('moment');
 
 function version(version) {
-  let first = version[0];
+  // The passed in version may or may not already be a string. See #1599.
+  const stringifiedVersion = '' + version;
```

Comment:
I code a lot in Java and we tend to avoid this, but I'm happy to use it here! 😄

Reply:
Yea… it's a matter of style. The language treats them as `let` variables, though if you want you can tell eslint not to let you reassign them. The advantage of reassigning them is that it prevents you from accidentally using the unmodified value, and I think reassigning them once at the top of the function is not so confusing. Though I could be convinced otherwise. 😁